plugins.txt

```diff
@@ -34,7 +34,7 @@ configuration-as-code:1.51
 credentials:2.5
 credentials-binding:1.27
 datadog:3.1.0
-ec2:1.62
+ec2:1.63
```

Comment:
https://github.com/jenkinsci/ec2-plugin/releases/tag/ec2-1.63 🚀 (and less open socket leaks \o/)